runtime/moonbase/tests/integration_test.rs

```diff
@@ -2838,7 +2838,7 @@ fn precompile_existence() {
 		let precompiles = Precompiles::new();
 		let precompile_addresses: std::collections::BTreeSet<_> = vec![
 			1, 2, 3, 4, 5, 6, 7, 8, 9, 1024, 1025, 1026, 2048, 2049, 2050, 2051, 2052, 2053, 2054,
-			2055, 2056, 2057, 2058, 2059, 2060, 2061,
+			2055, 2056, 2057, 2058, 2059, 2060, 2061, 2061, 2062, 2063, 2064,
```

Comment:
```suggestion
			2055, 2056, 2057, 2058, 2059, 2060, 2061, 2062, 2063, 2064,
```
probably happened during a merge into master